trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -159,7 +156,18 @@ def finalize(self):
             fuel.mass_properties.mass  = 0.
             configuration.fuel         = fuel	
 
-
+        # check if user specifies number of spanwise vortices
+        if self.settings.spanwise_vortices == None:
+            pass
+        else:
+            self.settings.discretization.defaults.wing.spanwise_vortices = self.settings.spanwise_vortices  
```

Comment:
There may be a better way to do these if statements / eliminate them. I am not sure how many times they are called but if called alot they could slow down the code.